functions-methods.js

```diff
@@ -20,6 +41,19 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(email){
+
+    if (email.includes("novi-education")) {return "Student"}
+    else if(email.includes("novi.nl")) {return"Medewerker"}
+    else if(email.includes("outlook.com" || "gmail.com")){return "Extern"}
+    return email
```

Comment:
Ik zou deze aanpassen naar een `else` i.p.v. `else if`. Als het geen student én geen medewerker is, dan is het wel een externe.